src/lt/plugins/clojure.cljs

```diff
@@ -295,6 +314,8 @@
                           (object/raise obj :editor.result (unescape-unicode (or (:result res) "")) loc)))))
 
 (behavior ::cljs-result.inline-at-cursor
+          :desc "Clojurescript: Show the resulting evaluation inline at cursor location"
+          :doc "This is probably the same as `::cljs-result.inline` but puts the result widget somewhere else."
```

Comment:
Please remove docs that aren't sure of functionality, here and other :doc keys that use "probably"